src/scenes/Authentication/ForgotPassword/ForgotPasswordForm.tsx

```diff
@@ -0,0 +1,71 @@
+import { Card, CardContent } from '@material-ui/core';
+import { Decorator, Mutator } from 'final-form';
+import React from 'react';
+import { Form, FormProps } from 'react-final-form';
+import { UserEmailInput } from '../../../api';
+import {
+  blurOnSubmit,
+  EmailField,
+  focusLastActiveFieldOnSubmitError,
+  SubmitButton,
+  SubmitError,
+} from '../../../components/form';
+
+export type ForgotPasswordFormProps = Pick<
+  FormProps<UserEmailInput>,
+  'onSubmit' | 'initialValues'
+> & { className?: string };
+
+export const ForgotPasswordForm = ({
+  className,
+  ...props
+}: ForgotPasswordFormProps) => (
+  <Form
+    {...props}
+    onSubmit={async (...args) => {
+      const res = await props.onSubmit(...args);
+      return res
+        ? {
+            ...res,
+          }
+        : undefined;
+    }}
+    decorators={decorators}
+    mutators={{ clearSubmitErrors }}
```

Comment:
These aren't necessary here since we have only one field that can never be invalid
```suggestion
```